Userland/Libraries/LibGL/SoftwareGLContext.h

```diff
@@ -223,6 +224,15 @@ class SoftwareGLContext : public GLContext {
         GLenum mode { GL_COMPILE };
     };
     Optional<CurrentListing> m_current_listing_index;
+
+    struct VertexAttribPointer {
+        GLint size = { 4 };
+        GLenum type = { GL_FLOAT };
```

Comment:
```suggestion
        GLint size { 4 };
        GLenum type { GL_FLOAT };
```

Let's be consistent :)